iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -160,11 +164,54 @@ namespace iroha_cli {
           local_time_, creator_, counter_, account_id, asset_id);
     }
 
+    nonstd::optional<iroha::model::Pager> InteractiveQueryCli::parsePager(
+        const std::string& encoded_tx_hash, const std::string& limit_str) {
+      return nonstd::make_optional<iroha::model::Pager>()
+        | [&encoded_tx_hash](iroha::model::Pager pager)
+            -> nonstd::optional<iroha::model::Pager> {
```

Comment:
Strange, can't the type be deduced here?

Reply:
Deduced type of `return nonstd::nullopt`
So adding this is needed.